src/crimson/os/alienstore/alien_store.h

```diff
@@ -21,6 +21,9 @@ class Transaction;
 namespace crimson::os {
 class AlienStore final : public FuturizedStore {
 public:
+  // number of cores that are PREVENTED from being scheduled
+  // to run alien store threads.
+  static constexpr int N_CORES_FOR_ALIENSTORE = 3;
```

Comment:
this is not part of the public interface. might want to move it to the .cc file.

Reply:
the variable name is misleading, it should be `N_CORES_FOR_SEASTAR`, as in this case, 3 cores are reserved for seastar.